activity_browser/bwutils/multilca.py

```diff
@@ -118,15 +118,16 @@ def __init__(self, cs_name: str):
                 "{} is not a known `calculation_setup`.".format(cs_name)
             )
 
-        if sum([v for rf in cs['inv'] for v in rf.values()]) == 0:
+        # check if all values are non-zero
+        if len([v for rf in cs['inv'] for v in rf.values() if v == 0]) != 0:
```

Comment:
Please check if we can refactor this in a more comprehensible way. One suggestion is to leave the len(..) != 0 statement out. The list will be false when there are no values in it anyway.